pkgs/development/python-modules/cylp/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, scipy
+, python
+, pytestCheckHook
+, cython
+, cbc
+, bzip2
+, zlib
+, pkg-config
+}:
+
+buildPythonPackage rec {
+  pname = "cylp";
+  version = "0.91.4";
+
+  src = fetchFromGitHub {
+    owner = "coin-or";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-m09s+QrkLGtF83tDLNjU4ECOqgW5RT0h9+fSwDeADag=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    cython
+  ];
+
+  buildInputs = [
+    cbc
+    bzip2
+    zlib
+  ];
+
+  propagatedBuildInputs = [
+    numpy
+    scipy
+  ];
+
+  preBuild = ''
+    # Leak some memory, reverting this commit. Otherwise there's a segfault
+    # that's being debugged in https://github.com/coin-or/CyLP/issues/105
+    substituteInPlace cylp/cy/CyClpSimplex.pyx --replace \
+      'del self.CppSelf' \
+      'pass'
+
+    # Rebuild all cython files
+    export CYLP_USE_CYTHON=1
+  '';
+
+  checkInputs = [ pytestCheckHook ];
+  pythonImportsCheck = [ "cylp" ];
+  pytestFlagsArray = [ "-s" ];
+
+  preCheck = ''
+    cd cylp/tests
+    rm __init__.py
+  '';
+
+  # https://github.com/coin-or/CyLP/commit/e5bab3cca81c0ad9886bcd8127923a436bcdaf9b
```

Comment:
How is this commit relevant to the disabled test file? I don't see anything about test_MIP at all.

Reply:
See comment below